$OpenBSD: patch-src_plugin_c,v 1.1 2017/09/04 10:54:20 dcoppa Exp $

fix crash on lyrics screen
(upstream git commit 36e3f4589afc5bf3d17c0584719a86a577340fa0)

Index: src/plugin.c
--- src/plugin.c.orig
+++ src/plugin.c
@@ -216,7 +216,7 @@ plugin_fd_add(struct plugin_cycle *cycle, struct plugi
 	p->data = g_string_new(NULL);
 	GIOChannel *channel = g_io_channel_unix_new(fd);
 	p->event_id = g_io_add_watch(channel, G_IO_IN|G_IO_HUP,
-				     plugin_data, cycle);
+				     plugin_data, p);
 	g_io_channel_unref(channel);
 }
 
